code/game/objects/items/food/moth.dm

```diff
@@ -0,0 +1,884 @@
+/obj/item/food/curd_cheese
+	name = "curd cheese"
+	desc = "Known by many names throughout human cuisine, curd cheese is useful for a wide variety of dishes."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "curd_cheese"
+	microwaved_type = /obj/item/food/cheese_curds
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3, /datum/reagent/consumable/cream = 1)
+	tastes = list("cream" = 1, "cheese" = 1)
+	foodtypes = DAIRY
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/cheese_curds
+	name = "cheese curds"
+	desc = "Not to be mistaken for curd cheese. Tasty deep fried."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "cheese_curds"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3)
+	tastes = list("cheese" = 1)
+	foodtypes = DAIRY
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/cheese_curds/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/dryable,  /obj/item/food/firm_cheese)
+
+/obj/item/food/herby_cheese
+	name = "herby cheese"
+	desc = "As a staple of mothic cuisine, cheese is often augmented with various flavours to keep variety in their diet. Herbs are one such addition, and are particularly beloved."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "herby_cheese"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3)
+	tastes = list("cheese" = 1, "herbs" = 1)
+	foodtypes = DAIRY | VEGETABLES
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/firm_cheese
+	name = "firm cheese"
+	desc = "Firm aged cheese, similar in texture to firm tofu. Due to its lack of moisture it's particularly useful for cooking with, as it doesn't melt easily."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "firm_cheese"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3)
+	tastes = list("aged cheese" = 1)
+	foodtypes = DAIRY | VEGETABLES
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/firm_cheese_slice
+	name = "firm cheese slice"
+	desc = "A slice of firm cheese. Perfect for grilling or making into delicious pesto."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "firm_cheese_slice"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3)
+	tastes = list("aged cheese" = 1)
+	foodtypes = DAIRY | VEGETABLES
+	w_class = WEIGHT_CLASS_SMALL
+	burns_on_grill = TRUE
+
+/obj/item/food/firm_cheese_slice/MakeGrillable()
+	AddComponent(/datum/component/grillable, /obj/item/food/grilled_cheese, rand(25 SECONDS, 35 SECONDS), TRUE, TRUE)
+
+/obj/item/food/mozzarella
+	name = "mozzarella cheese"
+	desc = "Delicious, creamy, and cheesy, all in one simple package."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "mozzarella"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3)
+	tastes = list("mozzarella" = 1)
+	foodtypes = DAIRY
+	w_class = WEIGHT_CLASS_SMALL
+
+//Base ingredients and miscellany, generally not served on their own
+/obj/item/food/grilled_cheese
```

Comment:
wait dont we already have grilled cheese?

Reply:
This is grilled cheese as in a grilled piece of cheese, a la halloumi, not a grilled cheese sandwich. The description quite clearly states that, I think.

Reply:
oh im dumb, carry on

Reply:
You bout to add moth halloumi???